RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -333,6 +339,9 @@ extension VoiceBroadcastPlaybackViewModel: VoiceBroadcastAggregatorDelegate {
     }
     
     func voiceBroadcastAggregatorDidEndLoading(_ aggregator: VoiceBroadcastAggregator) {
+        if state.playbackState != .stopped {
```

Comment:
We do something here only if the playbackState is buffering

(If the user Pause the playback while the aggregator was loading we should not play)